Makefile

```diff
@@ -73,15 +72,7 @@ release: dist
 	$(DEVENV)/bin/twine upload dist/*
 
 .PHONY: release-js
-release-js: clean check dev-js
-	$(eval current := $(shell npm view $(PROJECT) version))
-	$(eval package := $(shell npm version | grep $(PROJECT) | cut -d "'" -f 2))
-	@test $(current) != $(package) || ( \
-		echo cannot publish existing version $(current): update package.json; \
```

Comment:
You're not able to publish the same version again so this check was not necessary.